README.md

```diff
@@ -1,13 +1,44 @@
 # Sample Revel Applications
 
-Each directory contains a revel application.
+Each sub directory contains a standalone [Revel](http://revel.github.io/manual)  application, eg
+
+狂欢是好的，保存UTF8 。保持微笑，享受..抱怨..请让它好看;-)
+
 
 ```
 go get github.com/revel/samples
 revel run github.com/revel/samples/booking
 ```
 
-Visit http://revel.github.io for more info
+* [Booking](booking.html) 
+  - A database-driven hotel-booking application, including user management
+  
+* [Chat](chat.html) 
+  - A chat room demonstrating active refresh, long-polling (comet), and [websocket](http://revel.github.io/manual/websockets.html) implementations.
+  
+* [Validation](validation.html) 
+  - A demonstration of input [validation](http://revel.github.io/manual/validation.html).
+  
+* [Upload](upload.html) 
+  - Demonstrates single and multiple file uploads to a webserver via http.
+  
+* [Twitter OAuth](twitter-oauth.html) 
+  - Display `mentions` and allows `posting` to a Twitter account using OAuth.
+  
+* [Facebook OAuth2](facebook-oauth2.html) 
+  - Display Facebook user information using OAuth2.
+  
+  
+
+
 
 ## Contribute
-If you got a good example application then please make a pull request ;-)
+- If you spot and idiot mistake etc, then please make some noise
+- If you have a good sample application idea, then make some noise
+- If you got an app, then please make a pull request 
+- ta ;-)))))))))))))))
+- Early to bed, early to rise, makes a person, healthy and wise .. (code workz)
```

Comment:
Absolutely agree. We need to keep it professional. You can be goofy in chat, but this style of language is distracting from the purpose of the words. We're not trying to convey a friendly message, we're trying to teach. Keep it simple.